Automattic/portkey-android
# Edit screen: text edit dialog

This PR brings the edit text screen to be along the lines of the design.
Some things have been changed on purpose to make it easier to handle:
- color item size: `34dp` instead of the original `24dp` (too small to tap otherwise)
- margins around different color items: `12dp`
- margin for the whole horizontal list: `12dp`

![Screenshot_20190822-102301](https://user-images.githubusercontent.com/6597771/63518288-19173c80-c4c7-11e9-944f-716b756f5395.png)


To test:
1. open the app and take a picture, the edit mode screen will be shown
2. tap on the `Tt` icon
3. observe the textdialog with a placeholder text "your text here" appears, as in the picture.
4. verify the other editing controls on the top of the screen are not shown (sound, text, stickers, close)
5. verify you can change the color of text to any of the colors shown.
